cap/modules/reana/views.py

```diff
@@ -78,15 +80,18 @@ def create_workflow():
             "mc15_13TeV.123456.cap_recast_demo_signal_one.root"
         }
     }
-    access_token = current_app.config.get('REANA_ACCESS_TOKEN')
-    response = create_workflow_from_json(
-        workflow_json, name, access_token, parameters, workflow_engine)
 
     try:
         uuid = PersistentIdentifier.get('recid', record_id).object_uuid
+        experiment = CAPRecord.get_record(uuid)['_experiment']
+        access_token = current_app.config.get(
+            'REANA_ACCESS_TOKEN').get(experiment)
```

Comment:
Take care of the case when this experiment doesn't have a reana token (or simply experiment is None)